lab-brae/test/gallery-service-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+describe('Gallery Service', function() {
+  
+  beforeEach(() => {
+    angular.mock.module('cfgram');
+    angular.mock.inject(($rootScope, authService, galleryService, $window, $httpBackend) => {
+      this.$window = $window;
+      this.$rootScope = $rootScope;
+      this.authService = authService;
+      this.galleryService = galleryService;
+      this.$httpBackend = $httpBackend;
+    });
+  });
+
+  describe('galleryService.createGallery', () => {
+    it('should create a new gallery', () => {
+      let galleryData = {
+        name: 'example gallery',
+        desc: 'example description'
+      };
+
+      let headers = {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        Authorization: 'Bearer test token'
+      };
+
+      this.$httpBackend.expectPOST('http://localhost:8000/api/gallery', galleryData, headers)
```

Comment:
Change your localhost to your API environment variable like so: ${__API_URL__} to follow best practices.